website/playground/lang-rome-ast/.gitignore

```
@@ -0,0 +1,2 @@
+node_modules
+dist
```

So we know they are folders

```suggestion
node_modules/
dist/
```